src/megaclient.cpp

```diff
@@ -9913,6 +9913,15 @@ bool MegaClient::syncup(LocalNode* l, dstime* nds)
                                 RegCloseKey(hKey);
                             }
                         }
+
+                        lpath.append((char *)L":OECustomProperty", 34);
```

Comment:
Windows Search (enabled by default on Windows) adds this ADS to .eml and .pst files just after they are downloaded. Unfortunately that changes the modification time of the files too and Windows Search doesn't restore it.